airbyte-scripts/github-filler/main.py

```diff
@@ -0,0 +1,141 @@
+    return wrapper
+
+
+def get_headers(token: str):
+    return {
+        'Authorization': f'Bearer {token}',
+        'Accept': 'application/vnd.github.v3+json application/vnd.github.inertia-preview+json',
+    }
+
+
+def get_all_branches(rep, head):
+    url = f"https://api.github.com/repos/{rep}/branches"
+    response = requests.get(url=url, headers=head)
+    return response.json()
+
+
+def pull_requests(rep: str, branches: list, head: dict):
+    logging.info('Function: pull_requests')
+
+    url = f"https://api.github.com/repos/{rep}/pulls"
+
+    for branch in branches:
+        create_pr_data = {
+            'title': f'New PR from {branch.get("name")}',
+            'head': branch.get('name'),
+            'base': 'master'
+        }
+        # create PR
+        response = requests.post(url=url, data=json.dumps(create_pr_data), headers=head)
+        logging.info(f'Response status: {response.status_code}, response body: {"Success" if response.status_code in [200, 201] else response.content}')
+
+        # create review for PR
+        create_review_data = {
+            'body': f'Review commit for branch {branch.get("name")}',
+            'event': 'COMMENT',
+            }
+        review_url = f"https://api.github.com/repos/{rep}/pulls/{response.json().get('number')}/reviews"
+        response = requests.post(url=review_url, data=json.dumps(create_review_data), headers=head)
+        logging.info(f'Response status: {response.status_code}, response body: {"Success" if response.status_code in [200, 201] else response.content}')
+
+        # create comment for commit
+        create_comment_data = {
+            'body': f'comment for {branch.get("commit").get("sha")} branch',
+            }
+        commit_url = f"https://api.github.com/repos/{rep}/commits/{branch.get('commit').get('sha')}/comments"
+        response = requests.post(url=commit_url, data=json.dumps(create_comment_data), headers=head)
+        logging.info(f'Response status: {response.status_code}, response body: {"Success" if response.status_code in [200, 201] else response.content}')
+
+
+def add_issues_with_comments(rep: str, branches: List[dict], head: dict, labels: Optional[list], milestone: Optional[list], assignees: Optional[list]):
+    logging.info('Function: add_issues_with_comments')
+    url = f"https://api.github.com/repos/{rep}/issues"
+
+    for branch in branches:
+        data = {
+            'title': f'Issue for branch {branch.get("name")}',
+            'head': branch.get('name'),
+            'labels': labels,
+            'milestone': milestone,
+            'assignees': assignees
+        }
+
+        # add issue
+        response = requests.post(url=url, data=json.dumps(data), headers=head)
+        logging.info(f'Response status: {response.status_code}, response body: {"Success" if response.status_code in [200, 201] else response.content}')
+
+        # add issue comment
```

Comment:
this may be ok if we assumed to always add 1 comment to every issue.

If scenario will be changed, and we will need 0 or 2+ comments, this case does not suit for that,
It is better to create separate method/function `add_comment` and call it from here.